config/default.go

```diff
@@ -77,4 +77,5 @@ var defaultConfig = &Config{
 		Color:  "light-green",
 		Access: "rw",
 	},
+	GlobMatching: true,
```

Comment:
This makes sense I'll update.

Reply:
Not passing the config.Config down into the route package but that would require doubling up the public Lookup func as well as the matchingHosts func as that is where the glob is compiled.  If we keep the config being passed in we can apply the logic directly in the matchingHosts.   This would keep a single Lookup func and a single matchingHosts func.  Also we could add in some additional tests/bench_marks to test with/without glob matching.  That would also help with future comparisons of compiled globs/non-compiled globs/non-globs.    

I could also just set a local var and pass that along to the funcs to apply the logic but that would still require updated tests. 

Thoughts?

Reply:
Why not just pass a `bool` and default to `false` or also accept a `nil` config to simplify the tests?